test/com/google/javascript/jscomp/CollapsePropertiesTest.java

```diff
@@ -1617,92 +1917,91 @@ public void testLocalAliasOfEnumWithInstanceofCheck() {
         " var Fruit = null;\n" +
         " if (f == Enums$Fruit$APPLE) alert('apple');\n" +
         " if (f == Enums$Fruit$BANANA) alert('banana');\n" +
-        "}",
-        null);
```

Comment:
The merge conflict is caused by 76f458e596389ebba6927ea53f8073528b33e085. I can reverse that commit for now (it was just linter/formatting changes).